django/views/generic/base.py

```diff
@@ -71,12 +70,16 @@ def view(request, *args, **kwargs):
         view.view_class = cls
         view.view_initkwargs = initkwargs
 
-        # take name and docstring from class
-        update_wrapper(view, cls, updated=())
+        # Take special attributes as defined on the class, and annotations and
+        # any possible attributes set by decorators, e.g. @csrf_exempt, from
+        # the dispatch method. Note that __name__ and __qualname__ are left
+        # unchanged intentionally as view_class should be used to robustly
```

Comment:
```suggestion
        # the dispatch method. Note that __name__ and __qualname__ are
        # intentionally left unchanged, as view_class should be used to robustly
```
A slight change of word order here makes this read more naturally.

Reply:
Yes agreed